docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -155,49 +168,9 @@ Preparing For a Release
 #. Update the staging documentation.
    (For a documentation release ``${VERSION}`` should be the base release version in this step).
```

Comment:
To clarify, instead of exporting `VERSION`, the engineer can export `VERSION` and `BASE_VERSION` which will usually be the same. The release process can then get rid of this complication and use `BASE_VERSION` where necessary.